client/my-sites/comments/comment/comment-html-editor.jsx

```diff
@@ -0,0 +1,235 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { delay, map, reduce } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import AddImageDialog from 'post-editor/editor-html-toolbar/add-image-dialog';
```

Comment:
Okay I think this is causing our bundle size to jump 50%. Can we avoid the dialogs for now?

Reply:
This toolbar component is probably a candidate for an async bundle

Reply:
As a quick test you can try commenting these out and see what the new bundle stats look like.